gf-webapp-admin/src/component/rooms/room/Room.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import {IRoom} from "../../../model/room";
+import {ApartmentsType} from "./apartmentType/ApartmentType";
+import {Description} from "./description/Description";
+import {LongDescription} from "./longDescription/longDescription";
+import {Price} from "./price/Price";
+import {Promo} from "./promo/Promo";
+import {Rating} from "./rating/Rating";
+import styles from "./room.module.scss";
+import {Services} from "./services/Services";
+import {Slider} from "./slider/Slider";
+
+export const Room = ({room}: {room:IRoom}) => {
+  return (
+    <ul className={styles.room}>
```

Comment:
Why room has a "ul" tag element?  Is it a list? If not - fix it. (I think this shouldn't be a list)

Reply:
The "ul" tag has been changed to "div"